config/kubernetes/sig-docs/teams.yaml

```diff
@@ -44,6 +44,20 @@ teams:
     - zacharysarah
     - zhangxiaoyu-zidif
     privacy: closed
+  sig-docs-es-owners:
+    description: Admins for Spanish content
+    members:
+    - raelga
+    - alexbrand
+    privacy: closed
+  sig-docs-es-reviews:
+    description: PR reviews for Spanish content
+    members:
+    - raelga
```

Comment:
Here too.

The rest of the config looks good though 👍

Reply:
Thanks! I copy-pasted the original list sort by order of arrival.

Fixed in https://github.com/kubernetes/org/pull/685/commits/c17f48cf8233ca683d7dcdbd1c3400b55deac15a